src/platform/packages/shared/response-ops/alerts-table/components/alerts_table.tsx

```diff
@@ -351,6 +353,14 @@ const AlertsTableContent = typedForwardRef(
       }
     }, [alerts, columns, isLoadingAlerts, isSuccess, onLoaded, alertsCount]);
 
+    const fieldWithSortingError = useMemo(
+      () =>
+        queryParams.sort.find((sortField) =>
+          alertsError?.message?.includes(Object.keys(sortField)[0])
```

Comment:
I'm not sure this approach is ideal.

Just because there was an error where the field name is mentioned, it doesn't mean it is a **sorting** error, right?

I wish this api returned some kind of internal error code like `sorting_error` that we could also check for, but since that does not exist, could we maybe guarantee the error message has the word `sort` in it? Or maybe `sorting`? 

wdyt?